packages/camera/camera_android/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: camera_android
 description: Android implementation of the camera plugin.
 repository: https://github.com/flutter/packages/tree/main/packages/camera/camera_android
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+camera%22
-version: 0.10.4+1
```

Comment:
This can just be 10.4+2 since it's not changing any public API.

Reply:
Done